examples/haproxy/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+services:
+  neginx:
+    image: local/nginx 
+    ports:
+      - 40888:80
+    volumes:
+      - /matrix/static-files:/var/www/:rw
```

Comment:
Perhaps `ro` is enough?

Reply:
Yes, I also think it is enough. Good point.